net/raft/Makefile

```diff
@@ -1,28 +1,19 @@
 # The following commands may be useful for developing and debugging
 # our Raft packages. 
 
+
 build:
-	go install  -tags 'localhost_auth http_ok' chain/cmd/cored
+	go install -tags localhost_auth chain/cmd/cored
+	go install chain/cmd/corectl
 	dropdb --if-exists core0
-	rm -rf $(HOME)/.chaincore{0,1,2}
+	rm -rf $(HOME)/.chaincore?*
```

Comment:
Let's have this _just_ delete the raft directory, so that if we have certs in `.chaincore*` they won't get deleted.